src/adjoint/outputForward/adjointextra_d.f90

```diff
@@ -1458,4 +1458,1527 @@ subroutine sumdwandfw()
       end do
     end do
   end subroutine sumdwandfw
+
```

Comment:
these are AD generated code, so after you fix the corresponding fwd analysis code, you should re-run tapenade and see the changes here

Reply:
doing next